packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,143 @@
+
+![Client-based isolation for a unique IP address](/img/diagram1.png "Displays Bob as portal123 IP1 and Alice as portal123 IP2, both making authorize requests")
+
+#### Client-based isolation for users accessing `/authorize` from a NAT IP
+
+Alice, Bob, and Lisa all work from the same office. Since they access Okta through a Network Address Translation (NAT) IP from their office network, everyone shares the same IP address. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate-limiting framework creates a unique per minute request quota for the combination of every user's IP address, OAuth client ID of the application, and the device identifier set by Okta in each user's browser.
+
+![Client-based isolation for users accessing the authorize endpoint from a NAT IP](/img/diagram2.png "Displays Alice as portal123 Device1, Bob as portal123 Device2, Lisa as portal123 Device3, all making authorize requests that are a combination of the shared IP address")
+
+Alice: (NAT IP + portal123 + Device1 ID) Gets a quota of 500 total requests per minute and a maximum of two concurrent requests
+Bob: (NAT IP + portal123 + Device2 ID) Gets a quota of 500 total requests per minute and a maximum of two concurrent requests
+Lisa: (NAT IP + portal123 + Device3 ID) Gets a quota of 500 total requests per minute and a maximum of two concurrent requests
+
+> **Note:** When the requests are made using a non-browser client, the device cookie isn't present. In that case, such requests fall under a common quota with the device cookie being null (NAT IP + portal123 + null).
+
+#### Client-based isolation for users accessing `/authorize` through a proxy
+
+When `/authorize` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate-limiting framework to look for the IP address before the proxy to find the true client IP address.
+
+### Client-based rate-limiting modes
+
+The client-based rate-limiting framework can exist in one of three modes:
+
+| Mode               | Description                                                                            |
+| ------------------ | -------------------------------------------------------------------------------------- |
+| Disable            | Rate limits aren't enforced at the client-specific level. Rate limiting is based on the [org-wide Rate Limit](/docs/reference/rate-limits/#org-wide-rate-limits) values. |
+| Preview            | Rate limiting is still based on the [org-wide Rate Limit](/docs/reference/rate-limits/#org-wide-rate-limits) values, but client-specific rate limiting violation information is logged by firing System Log events. |
```

Comment:
This will change based on @ZoeMonosson-okta's final mocks

Reply:
Update mode names and rearrange the order (description is still the same):

1. Enforce and log per client (recommended): Rate Limiting is based on client-based rate-limiting values. Client-specific rate limiting violation information is logged by firing System Log events.

2. Log per client:  Rate limiting is still based on the org-wide Rate Limit values, but client-specific rate limiting violation information is logged by firing System Log events.

3. No action: Rate limits aren't enforced at the client-specific level. Rate limiting is based on the org-wide Rate Limit values.